web/client/components/geostory/contents/__tests__/ContentWrapper-test.jsx

```diff
@@ -54,56 +54,73 @@ describe('ContentWrapper component', () => {
         expect(el).toExist();
     });
     it('test classes generated from default props', () => {
-        const DEFAULT_THEME_CLASS_NAME = 'ms-bright';
-        const DEFAULT_ALIGN_CLASS_NAME = 'ms-align-center';
-        const DEFAULT_SIZE_CLASS_NAME = 'ms-size-full';
+        const DEFAULT_THEME_CLASS_NAME = '';
+        const DEFAULT_ALIGN_CLASS_NAME = ' ms-align-center';
```

Comment:
I think these changes are not necessary. It's not a problem to include them, but they make the test less readable.